js/filter.js

```diff
@@ -60,14 +60,23 @@ function renderSpeciesPage(speciesData, permalink) {
   specHead.innerHTML = speciesName.italics() + " " + specAuthority; 
   //specHead.appendChild(commonName);
   
-  var speciesCitation = document.createElement("p");
-  speciesCitation.innerHTML = "<b>Authority citation:</b> " + speciesData.authoritySpeciesCitation;
+  var speciesCitation = null;
```

Comment:
I wonder whether we should consider using jekyll templates instead of html generated in javascript. What do you think @JelleZijlstra ? Advantage of templates is that they are easier to edit for non-javascript programmers.

Reply:
Yes, templating would be better, but that's a bigger change than I have time for right now.

Reply:
I'd suggest to start migrating towards templating the next time we touch this filter.js file. In my experience, there is never enough time, and we'll save time down the road. Maintainability is key for enjoyment of working on the site in my option.

Reply:
Definitely a common refrain! Part of me wants to rewrite the whole thing in TypeScript.

Reply:
Great to hear your ideas on improving the code. I'd favor unit tests and templating over writing stuff in TypeScript.